soc/set_soc_prop.te

```diff
@@ -0,0 +1,22 @@
+# Rules for soc
+type set_soc_prop, domain;
+type set_soc_prop_exec, exec_type, file_type, vendor_file_type;
+init_daemon_domain(set_soc_prop)
+
+allow set_soc_prop proc_cmdline:file r_file_perms;
+allow set_soc_prop proc:file r_file_perms;
```

Comment:
Seem set_soc_prop only needs access /proc/cpuinfo, why it needs rules related with cmdlind and proc?